src/python/libcgroup.pyx

```diff
@@ -689,6 +689,27 @@ cdef class Cgroup:
 
         return current_path.decode('ascii')
 
+    @staticmethod
+    def move_proc(pid, dest_cgname):
```

Comment:
Do you think using `process` or `task` instead of `proc` make it more intuitive?

Reply:
Honestly I'm not sure.  We already have a python method, `get_procs()`, so I was trying to be consistent with that name.  What if I updated both:
* `move_procs()` -> `move_processes()`
* `get_procs()` -> `get_processes()`

The python multiprocessing library seems to use `process` fairly heavily
https://docs.python.org/3/library/multiprocessing.html#multiprocessing.current_process

Reply:
`move_process()` and `get_processes()` make sense. Feel free to merge the patchset with the minor change.